unitq/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "7781542f-b4a2-40e2-86cd-9987980a0ead",
+  "app_id": "unitq",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Harness the power of user feedback to improve product quality.",
+    "title": "unitQ",
+    "media": [],
```

Comment:
We highly encourage adding images here

Reply:
We plan on doing that in a later release. A number of our customers asked for this integration this week, so we would love to ship it before we start adjusting the details. I see that multiple integrations do not have any media. If possible, we would like to add it first and update the integration later. We are unfortunately already delayed according to our promise to multiple customers.